test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1

```diff
@@ -142,3 +142,29 @@ Describe "Tests for hashtable to PSCustomObject conversion" -Tags "CI" {
     }
 }
 
+Describe "Error message with settable Property information" {
+    BeforeAll {
+        Add-Type @"
+namespace HashtableConversionTest {
+    public class AType {
+        public string Name;
+        public string Path { get; set; }
+        public string Id { get; }
+    }
+}
+"@
+    }
+
+    It "Only settable properties are called out in the error message" {
```

Comment:
Shouldn't there be a test when there are no settable properties available?

Reply:
Test added.